src/Controls/src/Core/Compatibility/Handlers/ListView/Android/BaseCellView.cs

```diff
@@ -116,18 +119,26 @@ Element INativeElementView.Element
 			get { return _cell; }
 		}
 
-		public void SetAccessoryView(AView view)
+		public void SetAccessoryView(AView view, bool overrideAsAccessibilityElement = false)
 		{
 			if (AccessoryView != null)
 				RemoveView(AccessoryView);
+				_accessoryViewOverrideAsAccessibilityElement = false;
 
 			if (view != null)
 			{
 				using (var layout = new LayoutParams(ViewGroup.LayoutParams.WrapContent, ViewGroup.LayoutParams.MatchParent))
 					AddView(view, layout);
 
-				AccessoryView = view;
+				_accessoryViewOverrideAsAccessibilityElement = overrideAsAccessibilityElement;
+				if (overrideAsAccessibilityElement)
+					_mainText.LabelFor = view.Id;
+					_mainText.ImportantForAccessibility = ImportantForAccessibility.No;
```

Comment:
To @samhouts comment here

https://github.com/dotnet/maui/issues/6683#issuecomment-1144265619

I don't think this is correct behavior to set this to No

This makes it so accessibility only selects the switch and never the entire listview row. 

From testing inside the settings app the swipe behavior goes

- <Reads out description of list item cell> "double tap to activate"
- swipe right
- selects switch

It's important to have that first "double tap to activate" because the behavior might be that you want to double tap the cell and still select it.